hermes-pipeline/docs/action-data-mapping.md

```
@@ -270,33 +270,31 @@ This action triggers an internal `createProposal()` which emits PROPOSAL_CREATED
 
 #### EDITOR_ADDED / MEMBER_ADDED
 
-**Onchain (ZC16):**
+**Onchain:**
 - Action: `keccak256('GOVERNANCE.EDITOR_ADDED')` / `MEMBER_ADDED`
 - Topic: `bytes32(memberSpaceId)` - member's space ID (first 16 bytes)
-- Data: `abi.encode(address)` - 32 bytes with address in last 20 bytes
+- Data: empty
 
 **Proto Output:** `HermesRoleGranted`
 | Field | Source | Type |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)